cloud/scope/network_load_balancer_reconciler.go

```diff
@@ -37,6 +36,9 @@ func (s *ClusterScope) ReconcileApiServerNLB(ctx context.Context) error {
 		return err
 	}
 	if nlb != nil {
+		if nlb.LifecycleState != networkloadbalancer.LifecycleStateActive {
+			return errors.New("network load balancer is not in active state, hence reconciliation cannot happen")
```

Comment:
Thoughts about doing something like described in `load_balancer_reconciler.go`?

Reply:
fixed, please check..